pubspec.yaml

```diff
@@ -19,4 +19,4 @@ dependencies:
 dev_dependencies:
   flutter_test:
     sdk: flutter
-  pedantic: ^1.7.0
+  lints: ^2.0.1
```

Comment:
I think it should be `flutter_lint` here ?

Reply:
yes. thanks. I overlooked.